rpgJavaInterpreter-core/build.gradle

```diff
@@ -174,6 +174,16 @@ task fatJar(type: Jar) {
     with jar
 }
 
+//Creates smeup-rpg/rpgJavaInterpreter-core/build/libs/rpgJavaInterpreter-core-mute-all.jar
+task fatMuteJar(type: Jar) {
+    manifest {
+        attributes 'Main-Class': 'com.smeup.rpgparser.mute.StandalonemuterunnerKt'
```

Comment:
Why StandaloneMuteRunner is not capitalized?

Reply:
The file in which it resides is standalonemuterunner.kt, and so, as you obviously know, it's viewed as StandalonemuterunnerKt by Java.
I didn't capitalize the file name because I supposed that there is a convention for which capitalized files are just the ones contaning classes, not functions. Isn't it?

Reply:
Yes, true, I was thinking if it could make sense to call it:
"standalone_mute_runner.kt" or "standaloneMuteRunner.kt". The intent is making the name easier to read

Reply:
OK. I used @JvmName annotation